### PR TITLE
Unicycler: update v.0.50.0

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Changes https://github.com/rrwick/Unicycler/releases/tag/v0.5.0

### CODE REVIEW

File: tools/unicycler/unicycler.xml [hunk 1]
```diff
@@ -204,14 +198,27 @@ $pilon.no_pilon
             <param argument="--scores" type="text" value="3,-6,-5,-2" label="Comma-delimited string of alignment scores: match, mismatch, gap open, gap extend"/>
             <param argument="--low_score" optional="true" type="integer" value=""
                 label="Score threshold - alignments below this are considered poor" help="default = set automatically"/>
+            <param argument="--no_simple_bridges" type="boolean" truevalue="--no_simple_bridges" falsevalue="" checked="false" label="Simple long-read bridging" help="Default: No" />
         </section>
+        <param argument="--keep" type="select" label="Keep: level of file retention" help="Default: 1">
```
Comment (bgruening) [inline tools/unicycler/unicycler.xml:203]: the help is not needed here I think

Comment (bgruening) [reply tools/unicycler/unicycler.xml:203]: remove the colon in the label?

Comment (bgruening) [reply tools/unicycler/unicycler.xml:203]: Or reformat the lable to something like "outputs to keep"

File: tools/unicycler/unicycler.xml [hunk 2]
```diff
@@ -204,14 +198,27 @@ $pilon.no_pilon
...
+            <option value="0">0: only keep final files</option>
+            <option value="1" selected="true">1: also save graphs at main checkpoints</option>
```
Comment (bgruening) [inline tools/unicycler/unicycler.xml:205]: remove "also"?

**Review by bgruening — COMMENTED** [review]
just small nitpicks

File: tools/unicycler/unicycler.xml (hunk 2 above)
Comment (gallardoalba) [reply tools/unicycler/unicycler.xml]: Done, thanks!

**Review by bgruening — APPROVED** [review]

**Pull request merged**